README.md

```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
 * Return value
   - void
 
-## Advanced use
+## Advanced Use
+
+### Chain react of State
+
+The chain react is a common requirement, you can implement it by calling other Store in Store.
+
+#### Example
+
+Suppose you have a User Store, which records the number of tasks of the user. And a Tasks Store, which records the task list of the system. Every time a user adds a task, user's task number needs to be updated.
+
+```tsx
+// src/store/user
+export interface User {
+  name: string;
+  tasks: number;
+}
+export const user: User = {
+  dataSource: {
+    name: '',
+    tasks: 0,
+  },
+  async refresh() {
+    this.dataSource = await fetch('/user');
+  },
+};
+
+// src/store/tasks
+import { user }  from './user';
+
+export interface Task {
+  title: string;
+  done?: boolean;
+}
+export const tasks = {
+  dataSource: [],
+  async refresh() {
+    this.dataSource = await fetch('/tasks');
+  },
+  async add(task: Task) {
+    this.dataSource = await fetch('/tasks/add', task);
+
+    // Retrieve user information after adding tasks
+    await user.refresh();
+  },
+};
+
+// src/store/index
+import Icestore from '@ice/store';
+import { task } from './task';
+import { user } from './user';
+
+const icestore = new Icestore();
+const stores = icestore.registerStores({
+  task,
+  user,
+});
+
+export default stores;
+```
+
+#### Pay attention to circular calling
+
+Please pay attention to circular calling when call each other between Stores. 
```

Comment:
Please pay attention to circular dependencies problem when methods calling each other between stores.